.github/workflows/deploy.yml

```diff
@@ -0,0 +1,163 @@
+name: Deploy
+
+on:
```

Comment:
You may wanna add `workflow_dispatch` here for manual triggers too

Reply:
Can do, what would be the use case for a manual trigger?

Reply:
Just in case something went wrong during the push or to force a new deploy over at the target repos but I guess you can simply re-run the job if something goes wrong.